core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -141,6 +151,23 @@ class HttpGoogleServicesDAO(
         val insertMetadataBucket = getStorage(getBucketServiceAccountCredential).buckets.insert(serviceProject, metadataBucket)
         executeGoogleRequest(insertMetadataBucket)
     }
+
+    // unsafeRunSync is not desired, but return type for initBuckets dictates execution has to be happen immediately when the method is called.
+    // Changing initBuckets's signature requires larger effort which doesn't seem to worth it
+    createMetadataBucketNotification.unsafeRunSync()
+    createTopic.unsafeRunSync()
```

Comment:
you don't need to create the topic first?

Reply:
This was fixed at some point.